packaging/dbscripts/clusters_sp.sql

```diff
@@ -567,7 +568,33 @@ BEGIN
 
     IF FOUND THEN
         UPDATE cluster
-        SET upgrade_running = true
+        SET upgrade_running = true, upgrade_correlation_id = v_correlation_id
```

Comment:
Please reformat to have one column update per line

Reply:
Ack